app/src/main/java/com/pokemonshowdown/app/UpdateCheckTask.java

```diff
@@ -87,7 +87,10 @@ private boolean isUpdateAvailable(String currentVersion, String serverVersion) {
 
             if (server > current) {
                 return true;
+            } else if (current > server) {
+                return false;
```

Comment:
Add a line on top of this method with URL to our server plz, so that we have it on record